webpack/plugins/svg-compiler-plugin.js

```diff
@@ -0,0 +1,54 @@
+import fs from 'fs';
```

Comment:
There are existing loaders/plugins that do this as far as I'm aware. Also, do we need this and the Ajax request in `index.js`?